railties/test/application/test_runner_test.rb

```diff
@@ -430,6 +430,26 @@ class PostTest < ActiveSupport::TestCase
       end
     end
 
+    def test_unknown_option_is_invalid
+      create_test_file
+
+      assert_match 'invalid option: --unknown', run_test_command('--unknown')
+    end
+
+    def test_extra_plugin_options_goes_through_unscathed
+      skip 'cannot get this to load the plugin'
```

Comment:
Tried to trigger https://github.com/seattlerb/minitest/blob/61b74093b8194c857e1e9789672cce29dc1179c3/lib/minitest.rb#L91, which should search the load path as well (and `app_root/lib` should be in there).